.github/workflows/php-compatibility.yml

```
@@ -29,4 +32,4 @@ jobs:
         run: composer install
 
       - name: Run PHP Compatibility
-        run: vendor/bin/phpcs windows-azure-storage.php includes/ -p --standard=PHPCompatibilityWP --extensions=php --runtime-set testVersion 8.0-
+        run: vendor/bin/phpcs windows-azure-storage.php includes/ -p --standard=
```

Seems the last bits of this command were removed. I'm assuming that wasn't on purpose and should probably be added back
```suggestion
        run: vendor/bin/phpcs windows-azure-storage.php includes/ -p --standard=PHPCompatibilityWP --extensions=php --runtime-set testVersion 8.0-
```